### ISSUE TITLE
Add tests for non-tree phylogenetic networks

### ISSUE THREAD

**Issue by gaurav** [phyloref/phylo2owl#12]
`phylo2owl.py` currently writes out all the nodes found in the input tree file so it _should_ work with phylogenetic networks that aren't trees if DendroPy can parse them. If this is a necessary feature, we should add tests for this.

Requires #11, since I don't think DendroPy supports reticulate networks in Newick or Nexus, and I'm not sure if it supports it in NeXML.

**Comment by hlapp** [comment]
I agree this is lower priority, until we start validating against use-cases that require this.
